### PR TITLE
Add babel-runtime to fix ReferenceError: regeneratorRuntime

### LINKED ISSUE #39: ReferenceError: regeneratorRuntime is not defined with Create React App
I tried to use `react-pdf@2.0.0-beta` with Create React App and I got this error.

Steps to reproduce:
```bash
npm install -g create-react-app
create-react-app test-pdf
cd test-pdf
npm install react-pdf@next --save
```

```jsx
import React, { Component } from 'react';
import { Document, Page } from 'react-pdf'
import logo from './logo.svg';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <Document>
          <Page />
        </Document>
      </div>
    );
  }
}

export default App;
```

I tried to install `babel-polyfill` and requires it before `react-pdf` but it didn't solve the issue.
It's mentionned in the README that `babel-polyfill` is required by the library if we don't use it but I didn't find where in the code?

### PR DESCRIPTION
**Pull request by Kerumen**
After several tests I found the fix of this issue. It's needed to add the `babel-runtime` environment.

https://babeljs.io/docs/plugins/transform-runtime/

I tried to change the location of the `require('babel-polyfill')` but it didn't fix my issue. Anyway, it's a best practice to require it once at the top of your entry file.

Fixes #39 

PS: You shouldn't push your build files to GitHub :)

### CODE REVIEW

**Comment by wojtekmaj** [comment]
Hey, thanks so much for your PR!

What do you mean that I should not include built files? If I wanted people to use my library and the files were not prebuilt, wouldn't that require them to build them on their own, requiring them to have babel and stuff, which would make the install process way longer?

**Comment by Kerumen** [comment]
You should totally publish your built files on npm but not committing them on GitHub.
This way your commits and your repo are cleaner but when people install your library with npm they got everything they need.

File: package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
   "license": "MIT",
   "dependencies": {
     "babel-polyfill": "^6.23.0",
```
Comment (wojtekmaj) [inline package.json]: @Kerumen, shouldn't we get rid of babel-polyfill completely then?

**Comment by Kerumen** [comment]
@wojtekmaj I removed `babel-polyfill` and rebased on `new-architecture` branch.

**Comment by wojtekmaj** [comment]
Thank you very much for your input!

**Comment by tgmarinho** [comment]
I have the same problem, I read all links and posts above, but I don't see the solution for it.
Should I open a new issue?

**Comment by sjatkins** [comment]
Doing nothing to my code I now have this issue where I didn't before. So what the hell? And how is doing require of babel-polyfill meaningful in CRA environment where index.js is ES6 based?  I am confused.

**Pull request merged**